src/components/action-popover/action-popover-menu/action-popover-menu.component.tsx

```diff
@@ -1,38 +1,119 @@
 import React, { useCallback, useMemo, useContext } from "react";
-import PropTypes from "prop-types";
+import invariant from "invariant";
+
 import { Menu } from "../action-popover.style";
 import Events from "../../../__internal__/utils/helpers/events";
-import ActionPopoverItem from "../action-popover-item/action-popover-item.component";
+import ActionPopoverItem, {
+  ActionPopoverItemProps,
+} from "../action-popover-item/action-popover-item.component";
 import ActionPopoverDivider from "../action-popover-divider/action-popover-divider.component";
 import ActionPopoverContext from "../action-popover-context";
 
-const ActionPopoverMenu = React.forwardRef(
+type ActionPopoverChild =
```

Comment:
This same type is defined in `action-popover.component.tsx` - we should define it in one place, and import it into the other